@AH-64D Apache Longbow/Addons/fza_ah64_controls/config/cfgVehicles/animationSources.hpp

```diff
@@ -37,20 +37,19 @@ class AnimationSources
         initPhase = 1;
         animPeriod = 0.001;
     };
-    class msn_equip_american
+    class msn_equip_british
     {
-        displayName = "Install American Mission Equipment";
+        displayName = "Swap American Mission Equipment for British";
         author = "Apache mod development team";
-        onPhaseChanged = "";
+        forceAnimatePhase = 1;
+        forceAnimate[] = { "fcr_enable", 1};
+        onPhaseChanged = "if ((_this # 1) == 1) then {_this # 0 animatesource [""msn_equip_american"", 0];} else {_this # 0 animatesource [""msn_equip_american"", 1];};";
```

Comment:
Have you tried using the `forceAnimate` config field? https://community.bistudio.com/wiki/Arma_3:_Vehicle_Customization